draft-ietf-dnsop-aname.md

```diff
@@ -347,11 +347,24 @@ way as for normal (dynamic) updates.
 
 Sibling address records are served from authoritative servers with a
 fixed TTL. Normally this TTL is expected to be the same as the target
-address records' TTL (or the ANAME TTL if that is smaller); however
-the exact mechanism for obtaining the target is unspecified, so cache
-effects or deliberate policies might make the sibling TTL smaller.
+address records' TTL; however the exact mechanism for obtaining the
+target is unspecified, so cache effects, following ANAME and CNAME
+chains, or deliberate policies might make the sibling TTL smaller.
+
+This means that when adding A and AAAA records into the zone as a
+result of ANAME processing, the TTL to use is at most that of the
+TTL of the address target records. If you use a higher value,
+this will stretch the TTL which is undesired.
```

Comment:
but it could be also shorter? `min(ANAME_TTL, TARGET_TTL)`?

Reply:
It could be shorter, but the main reason is that the RRset is retrieved from a cache and its TTL was already decremented (what Tony calls a nearly-equal TTL).